roles/ewatercycle/templates/ewatercycle.yaml.j2

```diff
@@ -81,6 +81,7 @@ parameter_sets:
     directory: lisflood_global-masked_01degree
     config: lisflood_global-masked_01degree/settings_lisflood_ERA5.xml
     target_model: lisflood
+    supported_model_versions: !!set {2020.10: null}
```

Comment:
In the code for lisflood it says:
`available_versions = ("20.10",)`

Reply:
https://github.com/eWaterCycle/ewatercycle/blob/c794ef752215bc8a663d87aab2374f68e0559b3d/src/ewatercycle/models/lisflood.py#L37